UpnetixSystemAlertQueue/Classes/AlertFactoryManager.swift

```diff
@@ -24,16 +24,25 @@ open class AlertFactoryManager {
     ///   - preferredStyle: Style of the alert controller. `.alert` by default.
     ///   - priority: priority (by default it's 0)
     ///   - withActions: UIalert actions
+    ///   - textFieldAttributes: Optional textField attributes
     @discardableResult
     open func presentActionAlert(withMessage message: String?,
                                  title: String?,
                                  preferredStyle: UIAlertController.Style = .alert,
                                  priority: Int = 0,
-                                 withActions: [UIAlertAction]) -> UIAlertController {
+                                 withActions: [UIAlertAction],
+                                 textFieldAttributes: TextFieldAttributes? = nil) -> UIAlertController {
```

Comment:
With this implementation you cannot add more than 1 textField. Is this expected behaviour?

Reply:
I agree that it should be possible to add more than one textfield, and I think I've discussed that with Iliyan , and the idea was to use an array of TextFieldAttributes, with one for each textfield we want. Were there a problem with that @iliyanivanov96

Reply:
Fixed